pkg/scaffold/controller/controller.go

```diff
@@ -251,10 +251,7 @@ func (r *Reconcile{{ .Resource.Kind }}) Reconcile(request reconcile.Request) (re
 	err = r.Get(context.TODO(), types.NamespacedName{Name: deploy.Name, Namespace: deploy.Namespace}, found)
 	if err != nil && errors.IsNotFound(err) {
 		log.Info("Creating Deployment", "namespace", deploy.Namespace, "name", deploy.Name)
-		err = r.Create(context.TODO(), deploy)
-		if err != nil {
-			return reconcile.Result{}, err
-		}
+		return reconcile.Result{}, r.Create(context.TODO(), deploy)
```

Comment:
Though this is more concise, but has poor readability. I had to search where did `Create` go :)

I think the previous version is more readable.
```Go
err = r.Create(....)
return reconcile.Result{}, err
```

Reply:
Agreed and thanks.